### DISCUSSION TITLE
Display download speed

### DISCUSSION THREAD

**Discussion by nickshanks347** [abraunegg/onedrive#2418] [Q&A] [answered]
Is there a way of showing download speed?

For example:
```
Downloading   5% |oo                                      |   ETA   00:00:38   |   18MB/s
```

If not, could this be implemented?

**Comment by abraunegg** [comment, marked as answer]
@nickshanks347 
> Is there a way of showing download speed?

No there is not a way to show this at present.

> If not, could this be implemented?

Please open a Feature Request. I may look at this for v2.5.0, however given that logging is all over the place at present, it probably wont get looked at till a much later release.
